### PR TITLE
Adds Catalan Translation

### PR DESCRIPTION
**Pull request by 6temes**
### Summary

This PR adds the locale for Catalan language. 

Related #771

### CODE REVIEW

**Comment by GBH** [comment]
Nice. Don't forget to add it here: https://github.com/comfy/comfortable-mexican-sofa/blob/master/lib/comfortable_mexican_sofa/configuration.rb#L92

**Comment by GBH** [comment]
Also... You don't need to install CMS in a new project to test your stuff. CMS can be run as a stand-alone for development purposes. Just run migrations and `rails s` it.

It's described in https://github.com/comfy/comfortable-mexican-sofa/blob/master/CONTRIBUTING.md

**Comment by 6temes** [comment]
Sorry for the delay, @GBH . I just finished the changes.

Please, tell me if everything is Ok.

I created a PR for comfy-blog as well.

File: config/locales/ca.yml [hunk 1]
```diff
@@ -0,0 +1,264 @@
+# encoding: utf-8
```
Comment (GBH) [inline config/locales/ca.yml]: no need for this header anymore. remove it please

**Comment by GBH** [comment]
Merged. Thanks!

**Pull request merged**